### PR TITLE
Add release notes for Kyma 1.25

### PR DESCRIPTION
**Pull request by majakurcius**
**Description**

Changes proposed in this pull request:

- Add release notes for Kyma 1.25

**Related issue(s)**
#796

### CODE REVIEW

**Comment by a-thaler** [comment]
We decided to update the minor version of fluent-bit to version 1.8.x
Also we decided to make a minor upgrade of grafana to 7.5.x
That is worth to mention, could you add sections for it

**Comment by majakurcius** [comment]
@a-thaler Done. Together with the OAuth2 Proxy, which I was instructed to add via DM.

File: content/blog-posts/2021-12-08-release-notes-1.25/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  - release-notes
+type: release
+releaseTag: "1.25.0"
+redirectFrom:
+  - "/blog/release-notes-125"
+---
+
+Salve!
+If you're wondering why the salutation in Latin and why we're using the _lingua franca_ at the moment:
+after we left Berlin, we decided to stop exploration for just a little while and land on this unnamed island to patch our boat a bit and boost our security. This release is not big, but it's an important one, as it brings you upgraded Istio and updated Ory stack, as well as upgraded Observability components.
```
Comment (NHingerl) [inline content/blog-posts/2021-12-08-release-notes-1.25/index.md:15]:
```suggestion
After we left Berlin, we decided to stop exploration for just a little while and land on this unnamed island to patch our boat a bit and boost our security. This release is not big, but it's an important one, as it brings you upgraded Istio and updated Ory stack, as well as upgraded Observability components.
```
Capitalized if it's 2 or more sentences?

Comment (majakurcius) [reply content/blog-posts/2021-12-08-release-notes-1.25/index.md]: After discussing it offline, the majority of TWs voted against capitalizing it, so I'm going to leave it as is.

**Comment by majakurcius** [comment]
The release has been cancelled, so I'm closing the PR.

**Pull request closed without merging**